plugin-server/src/cdp/fetch-executor.ts

```diff
@@ -120,8 +121,9 @@ export class FetchExecutor {
 
             resParams.response = {
                 status: fetchResponse.status,
-                body: responseBody,
+                headers: Object.fromEntries(fetchResponse.headers.entries()),
             }
+            resParams.body = responseBody
```

Comment:
The body is a level higher because even in failure cases it's possible to get a body back (some 500s or 400s get a body, those are "failed" fetches, but they might still have useful info for the user)

Reply:
The tests really don't like this change but I need to figure out if it is due to the test setup now being invalid or if this has actually broken something...

Reply:
Okay the main issue was the mock data in the test so this should be fine 🤔